lstchain/io/event_selection.py

```diff
@@ -64,13 +77,105 @@ class DL3FixedCuts(Component):
         default_value=[1],
     ).tag(config=True)
 
-    def gh_cut(self, data):
-        return data[data["gh_score"] > self.fixed_gh_cut]
+    def apply_global_gh_cut(self, data):
+        """
+        Applying a global gammaness cut on a given data
+        """
+        return data[data["gh_score"] > self.global_gh_cut]
+
+    def energy_dependent_gh_cuts(
+        self, data, energy_bins, min_value=0.1,
+        max_value=0.99, smoothing=None, min_events=10
+    ):
+        """
+        Evaluating energy-dependent gammaness cuts, in a given
+        data, with provided reco energy bins, and other parameters to
+        pass to the pyirf.cuts.calculate_percentile_cut function
+        """
+        # To be sure to have the efficiency in between 0 and 100
+        if self.gh_efficiency < 1:
+            self.gh_efficiency *= 100
```

Comment:
I don't understand this. A setting is being changed without even reporting it. Surely less than 1% efficiency is a strange choice, but one would try it during some test, and would end up inadvertently with a much larger efficiency.

Reply:
No, the values in the config is entered in %, [0,1), and it is just multiplied by 100 to use in the function.

Reply:
Than multiply it *always* by 100 **without** changing the value stored in the class.

Reply:
Ok.

Reply:
I change the value to be provided in the %, [0, 100], and no need to multiply/change the value in the class.